core/creators/inline.js

```diff
@@ -35,11 +35,17 @@
 			return null;
 		}
 
-		var editor = new CKEDITOR.editor( instanceConfig, element, CKEDITOR.ELEMENT_MODE_INLINE ),
-			textarea = element.is( 'textarea' ) ? element : null;
+		var textarea = element.is( 'textarea' ) ? element : null,
+			editorData = textarea ? textarea.getValue() : element.getHtml();
```

Comment:
To be completely honest, at some point I need to pass `editorData` to `delayCreationOnDetachedElement`\- so I need `if` between var declarations, and need to extract `editorData`. After some changes, I left this extraction, so that both below `if...else` branches look more alike. But it will work in the old way...

Reply:
In this comment I've only meant that previously we just used `textarea.getValue()`, and now if it is not present we fall back to `element.getHtml()` instead - and my question was how did it work previously if the element was not a textarea?